Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -170,6 +170,7 @@ export function runQTest(args: QTestArguments): Result {
         Cmd.flag("--enableVsJitDebugger", Environment.hasVariable("[Sdk.BuildXL]enableVsJitDebugger")),
         Cmd.flag("--qTestIgnoreQTestSkip", args.qTestIgnoreQTestSkip),
         Cmd.option("--qTestAdditionalOptions ", args.qTestAdditionalOptions, args.qTestAdditionalOptions ? true : false),
+        Cmd.option("--envVarsToKeep ", args.envVarsToKeep || ""),

```

Comment:
BuildXL already limits the environment variables visible to its child processes. Is an additional QTest restriction necessary? I worry about adding configuration complexity here. What would be the harm of allowing all of the DBS.QTest.exe pip's environment variables to pass through? Maybe you can provide an example to illustrate the use case

Reply:
Suresh has a PR that will allow all of the pip environment variables through. Let me abandon this PR for now.